### PR TITLE
Fix no follow  (#75)

### PR DESCRIPTION
**Pull request by Shaun-Fong**

### CODE REVIEW

**Comment by jcs090218** [comment]
Thanks for working on this! ❤️ 👍 

The code looks good to me, however, I think the code can be simpler! I think is possible to avoid `try ... catch ...` block and just test to see if the QR code page exists, if it does exists we do the operations. Otherwise, we simply ignore and jumps to:

https://github.com/game-ci/unity-license-activate/blob/ed9ae7a93dff45b140bc01bf8eb3e27522ea0c47/lib/activate.js#L53

The code should be somewhat similar to:

```js
console.log('Check to skip "QR Code" page...');

const needQRCode = await page.waitForSelector(".g6.connect-scan-group");

if (needQRCode) {
  console.log('[INFO] Start skipping QRCode page...');

  let loginBtn = await page.$('a[rel="nofollow"]');
  loginBtn.click();

  // do the rest to click on Account button!
  // ...
}
```

Do you think we can do it this way? Thanks! ;)

**Comment by Shaun-Fong** [comment]
> Do you think we can do it this way? Thanks! ;)

yep you are right. The code shouldn't be written in catch , I'll update it.

**Comment by Shaun-Fong** [comment]
@jcs090218 everything works fine now.

File: lib/activate.js [hunk 1]
```diff
@@ -50,6 +50,32 @@ async function start(email, password, alf, verification, emailPassword) {
...
+      //can't click by DOM API
+      let emailLogin = await page.$('a[data-event="toMailLogin"]');
+
+      let aBox = await emailLogin.boundingBox();
+
+      await page.mouse.move(aBox.x + 100, aBox.y + 25);
+      await page.mouse.down();
+      await page.mouse.up();
+    } else {
+      console.log("[INFO] Skip 'Check Follow'...");
```
Comment (jcs090218) [inline lib/activate.js:76]: I prefer to print a log in the `if` clause, and get rid of the `else` clause. That way we are able to debug everything step by step. For example:

```js
await page.waitForNavigation();

const needFollowJump = await page.$('.g6.connect-scan-group');

// --- Assumed above operations cause error

if (needFollowJump) {
  // With the log, we can identify the error immediately!
  //console.log("[INFO] Skipping 'Check Follow'...");

  // Or below operations cause error, we have no idea.
  let loginBtn = await page.$('a[rel="nofollow"]');
}
```

WDYT?

**Review by jcs090218 — COMMENTED** [review]
Small nit.

Other looks good! ;)

File: lib/activate.js (hunk 1 above)
Comment (Shaun-Fong) [reply lib/activate.js]: 😉

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
LGTM! Thank you for your contribution! ❤️ 🚀

**Pull request merged**